news/2020-19-11-mappers-report-september-october.md

```diff
@@ -0,0 +1,211 @@
+
+### Christmas Queue 2020
+
+The time honored Christmas Queue is back for its 10th year running! The Christmas Queue is a collective community effort to help mod and rank beatmaps related to the holiday and winter season. Now with a discord server and website to help keep the project organized, the organizers hope to help bring as much holiday cheer to the osu! mapping community as possible. Like before, in order to make this a great success again, the Christmas Queue is in need of motivated individuals to help push things forward. If you are a modder or mapper interested in contributing to this holiday event, be sure to check it out!
+
+You can find the information thread which includes further links and more [here](https://osu.ppy.sh/community/forums/topics/1173347?n=1)!
+
+### Community Guides
+
+This month, we have several mapping and modding related guides to spotlight! The knowledge base the mapping community makes and shares with each other grows ever bigger, and we hope that this can help you find some cool ideas or new content to learn from.
+
+The first is a new YouTube video by the experienced player, mapper, modder, and NAT member [Seto Kousuke](https://osu.ppy.sh/users/2857314)! In his video [5 advices for newbie mappers](https://www.youtube.com/watch?v=zrvfYtYaIAA) he goes over some tips from his experienced perspective that may not be intuitive for beginner mappers to learn, and watching it can be a great way for aspiring mappers to start getting pointed in the right direction in their mapping journey.
+
+The next is a pair of recent videos from long time mapper and Beatmap Nominator [Sonnyc](https://osu.ppy.sh/users/11771). The first, [Making Symmetric Sliders](https://www.youtube.com/watch?v=nYkMFvq4ZLU) serves to quickly show you how to make symmetrical sliders quickly. Even if you don't map symmetrically, knowing how to make balanced sliders to manipulate later can be hugely helpful. The second, [Increasing Mapping Speed](https://www.youtube.com/watch?v=sMKJ9H-qGg0) goes over an experience nearly every mapper has had - mapping slowly at first, and then later how to improve on putting your ideas out there quickly. Other videos on Sonnyc's channel can be super helpful guides to mappers new and old alike, so be sure to check them out!
+
+Moving away from motivation, time taken, and visuals, we have a [guide all about hitsounding](https://hackmd.io/@Net0/HitsoundsGuide) by [Net0](https://osu.ppy.sh/users/5099768)! This guide about hitsounds is incredibly in depth, going all the way from how to name your files to theory, choosing samples, using the editor, and how hitsounding relates back to actual instruments in the songs we love to map and play. If you've ever found hitsounding beatmaps to be intimidating, this guide will definitely help to clear up a lot of questions about it you may have.
```

Comment:
```suggestion
Moving away from motivation, time taken, and visuals, we have a [guide all about hitsounding](https://hackmd.io/@Net0/HitsoundsGuide) by [Net0](https://osu.ppy.sh/users/5099768)! This guide about hitsounds is incredibly in-depth, going all the way from how to name your files to theory, choosing samples, using the editor, and how hitsounding relates back to actual instruments in the songs we love to map and play. If you've ever found hitsounding beatmaps to be intimidating, this guide will definitely help to clear up a lot of questions about it you may have.
```
'time taken' looks a little weird there, not sure if I understood what that means